src/lib-rust/src/logging.rs

```diff
@@ -0,0 +1,161 @@
+#![allow(unused_variables)]
+
+use crate::locator::{LocationContext, VelopackLocator, VelopackLocatorConfig};
+use std::path::PathBuf;
+
+#[cfg(not(target_os = "windows"))]
+use std::path::Path;
+
+#[cfg(feature = "file-logging")]
+use simplelog::*;
+#[cfg(feature = "file-logging")]
+use time::format_description::{modifier, Component, FormatItem};
+
+#[cfg(target_os = "linux")]
+fn default_file_linux() -> PathBuf {
+    Path::new("/tmp/velopack.log").to_path_buf()
+}
+
+#[cfg(target_os = "macos")]
+fn default_file_macos() -> PathBuf {
+    #[allow(deprecated)]
+    let user_home = std::env::home_dir();
+
+    if let Some(home) = user_home {
+        let mut lib_logs = home.clone();
+        lib_logs.push("Library");
+        lib_logs.push("Logs");
+
+        if lib_logs.exists() {
+            lib_logs.push("velopack.log");
+            return lib_logs;
+        }
+    }
+
+    return Path::new("/tmp/velopack.log").to_path_buf();
+}
+
+/// Default log location for Velopack on the current OS.
+#[allow(unused_variables)]
+pub fn default_logfile_from_locator(locator: VelopackLocator) -> PathBuf {
+    #[cfg(target_os = "windows")]
+    {
+        return locator.get_root_dir().join("Velopack.log");
```

Comment:
I realize NTFS is often case insensitive, but the C# side does a lowercase file name. For consistency it would be nice to have it match.
```suggestion
        return locator.get_root_dir().join("velopack.log");
```